### PR TITLE
sql/postgres: finish handling comments creation/modification for indexes, tables and columns

### PR DESCRIPTION
**Pull request by a8m**

### CODE REVIEW

File: sql/postgres/migrate.go [hunk 1]
```diff
@@ -166,28 +166,43 @@ func (s *state) dropTable(drop *schema.DropTable) {
 	})
 }
 
-// modifyTable builds and executes the queries for bringing the table into its modified state.
+// modifyTable builds the statements that bring the table into its modified state.
```
Comment (masseelch) [inline sql/postgres/migrate.go:169]:
```suggestion
// modifyTable builds the statements that carry the table into its modified state.
```

File: sql/postgres/migrate.go [hunk 2]
```diff
@@ -166,28 +166,43 @@ func (s *state) dropTable(drop *schema.DropTable) {
...
 			dropI = append(dropI, change.I)
 		case *schema.ModifyIndex:
+			k := change.Change
+			if change.Change.Is(schema.ChangeComment) {
+				from, to, err := commentChange(sqlx.CommentDiff(change.From.Attrs, change.To.Attrs))
+				if err != nil {
+					return err
+				}
+				comments = append(comments, s.indexComment(modify.T, change.To, to, from))
+				// If only the index comment was changed.
```
Comment (masseelch) [inline sql/postgres/migrate.go:201]:
```suggestion
				// If only the index' comment was changed.
```

**Review by masseelch — APPROVED** [review]

File: sql/postgres/migrate.go (hunk 1 above)
Comment (a8m) [reply sql/postgres/migrate.go:169]: Not a native speaker, but not sure that means the same. Am I wrong?

Comment (a8m) [reply sql/postgres/migrate.go:169]: cc @rotemtam

File: sql/postgres/migrate.go (hunk 2 above)
Comment (a8m) [reply sql/postgres/migrate.go:201]: 👍

**Pull request merged**